src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -219,7 +219,7 @@ uct_rc_mlx5_iface_common_poll_rx(uct_rc_mlx5_iface_common_t *mlx5_common_iface,
 static UCS_F_ALWAYS_INLINE void
 uct_rc_mlx5_common_post_send(uct_rc_iface_t *iface, enum ibv_qp_type qp_type,
                              uct_rc_txqp_t *txqp, uct_ib_mlx5_txwq_t *txwq,
-                             uint8_t opcode, uint8_t opmod, unsigned sig_flag,
+                             uint8_t opcode, uint8_t opmod, uint8_t fm_ce_se,
```

Comment:
What does `fm_ce_se` mean?

Reply:
shortcut for "fence mode"+"completion event"+"solicited event" (the u8 var has all three fields as defined in PRM)